frontend/js/containers/QGis.jsx

```diff
@@ -175,24 +175,16 @@ const QGis = React.createClass({
         let ids = features.map((f) => {
             return f.id;
         }).join(',');
-        if (window.VALAMB && window.VALAMB.viewOnMapById) {
-            window.VALAMB.viewOnMapById(`'${this.props.featureType}',"${ids}"`);
-        }else {
         /*eslint-disable */
-            console.log(`VALAMB.viewOnMapById('${this.props.featureType}',"${ids}")`);
+        VALAMB.viewOnMapById(`'${this.props.featureType}',"${ids}"`);
```

Comment:
Do we always have a mock VALAMB object, so that it doesn't break when the real VALAMB  object is not there?